### DISCUSSION TITLE
How to use enhanced:img with multiple images/srcset

### DISCUSSION THREAD

**Discussion by lrwagner** [sveltejs/kit#12660] [Q&A]
Say I have something like this to render a hero img:
```svelte
	<enhanced:img
		src="$lib/assets/imgs/hero.jpg"
		sizes="(min-width: 1920px) 1920px, 
                             (min-width: 1440px) 1440px, 
                             (min-width: 1024px) 1024px, 
                             (min-width: 768px) 768px, 
                             100vw"
		alt="Hero"
	/>
```
What if I wanted to show a different img on Mobile because I want a different crop or aspect ratio - or maybe even a completely different img? Passing additional images to srcset doesn't seem to work.

**Comment by dubfonik** [comment]
I'm surprised this isn't an option (yet).

**Comment by teemingc** [comment]
I've created an issue for this here https://github.com/sveltejs/kit/issues/13275
